launch.go

```diff
@@ -84,7 +84,7 @@ func redirectURL(w http.ResponseWriter, r *http.Request) {
 func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
 	hostURL := settings.Get("SURVEY_RUNNER_URL")
 	surveyURL := r.URL.Query().Get("url")
-	log.Println("Request Received: "+ hostURL + "?url=" + surveyURL)
+	log.Println("Request Received: " + hostURL + "?url=" + surveyURL)
```

Comment:
We shouldn't be reconstructing the full URL for the log message (also, it's missing the `quick-launch` bit). I think it's enough here to just log the surveyURL, something like:
```
log.Println("Quick launch request received", surveyURL)
```